src/pages/regions/index.tsx

```diff
@@ -96,8 +97,11 @@ const RegionsPage: FC<Props> = ({
 
 export default RegionsPage
 
-export const pageQuery = graphql`
-  query RegionsQuery {
+export const query = graphql`
+  query MyQuery($id: String) {
```

Comment:
suggestion: markdown query not in use, can be removed

Reply:
removed the markdown query